cmd/osctl/cmd/kubeconfig.go

```diff
@@ -8,41 +8,55 @@ package cmd
 import (
 	"fmt"
 	"os"
+	"path/filepath"
+	"sync"
 
 	"github.com/spf13/cobra"
-	"google.golang.org/grpc/metadata"
 
 	"github.com/talos-systems/talos/cmd/osctl/pkg/client"
 	"github.com/talos-systems/talos/cmd/osctl/pkg/helpers"
 )
 
 // kubeconfigCmd represents the kubeconfig command
 var kubeconfigCmd = &cobra.Command{
-	Use:   "kubeconfig",
+	Use:   "kubeconfig [local-path]",
 	Short: "Download the admin kubeconfig from the node",
-	Long:  ``,
+	Long: `Download the admin kubeconfig from the node.
```

Comment:
nit: I think I'd prefer to be explicit here and skip the default to give it the same feel as `osctl cp`. But not gonna block. My OCD will get me enough later to where I can change in a follow up.